src/App.tsx

```diff
@@ -1,61 +1,119 @@
+import React, { useState } from 'react';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Tobo';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const findUserById = (userId: number) => {
+  return usersFromServer.find((user) => user.id === userId) || null;
+};
+
+const todosArr: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: findUserById(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [selectUser, setSelectUser] = useState('0');
```

Comment:
```suggestion
  const [selectedUserId, setSelectedUserId] = useState('0');
```
It would be more correct